src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php

```diff
@@ -222,4 +231,52 @@ private function getMatchAndPosition(AbstractString $text, string $regex): array
 
         return $output;
     }
+
+    private function uploadMedia(array $media): array
+    {
+        $pool = [];
+
+        foreach ($media as [
+             'file' => $file,
+             'description' => $description,
+        ]) {
+            $pool[] = [
+                'description' => $description,
+                'response' => $this->client->request('POST', sprintf('https://%s/xrpc/com.atproto.repo.uploadBlob', $this->getEndpoint()), [
```

Comment:
there is currently no error handling

Reply:
Error is managed by thoses lines in the next loop

https://github.com/symfony/symfony/blob/8fedc9b60d80711a808a7e6ab3d7face9dd30aa1/src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php#L264-L266